recipes/rauth/meta.yaml

```diff
@@ -0,0 +1,31 @@
+package:
```

Comment:
Please use jinja templating variables as in the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L3-L6